src/System.Runtime.InteropServices.RuntimeInformation/src/RuntimeInformation.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Diagnostics;
+
+namespace System.Runtime.InteropServices
+{
+    public static partial class RuntimeInformation
+    {
+        private const string RuntimeName = ".NET Core";
+        private static Architecture? s_processArch = null;
+        private static readonly bool s_is64BitProcess = IntPtr.Size == 8;
+
+        public static string FrameworkDescription
+        {
+            get
+            {
+                return RuntimeName;
```

Comment:
@weshaggard @davidfowl Yes i understood that we want to put a version in the string, but it's undecided whether to give the file version or the nuget version (with beta signatures) or the framework marketing (4.6, etc) version. We don't have a format defined. But if the requirement is to just show some number, then why is version needed if it's not adding meaning.